src/core/frames/frame_controller.ts

```diff
@@ -152,13 +152,16 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
   }
 
   requestFinished(request: FetchRequest) {
-    this.element.removeAttribute("busy")
+    this.element.removeAttribute("aria-busy")
   }
 
   // Form submission delegate
 
   formSubmissionStarted(formSubmission: FormSubmission) {
-
+    const frame = this.findFrameElement(formSubmission.formElement)
+    if (frame) {
```

Comment:
Is `frame` ever *not* present? Looks like it falls back to `this.element`: https://github.com/hotwired/turbo/blob/0c2bd1446fb202eed2a234ba1ccc7250bedba950/src/core/frames/frame_controller.ts#L220-L222

Reply:
@javan ah, I must've missed this while splitting out the separate `[aria-busy]` PR from the original: https://github.com/hotwired/turbo/pull/156/files#diff-8f44fafd854f6f716d51a2b05b5689266a9b66e0b1ad9d68570bd4e8cefd0a47R161